tests/JSONSerializeTest.cs

```diff
@@ -1,4 +1,20 @@
-﻿using System;
+﻿﻿/// <summary>
```

Comment:
I doubt there is value in having license headers in the doc :-)

let's do like:
https://github.com/apache/npanday/blob/trunk/dotnet/assemblies/NPanday.Utils/src/main/csharp/GacUtility.cs#L2-L19


On Thu, Mar 12, 2015 at 11:47 AM, Erik Jan de Wit <notifications@github.com>
wrote:

> In tests/JSONSerializeTest.cs
> <https://github.com/aerogear/aerogear-unifiedpush-dotnet-client/pull/2#discussion_r26291861>
> :
>
> > @@ -1,4 +1,20 @@
> > -﻿using System;
> > +﻿﻿/// <summary>
>
> depends do we want it to end up in the doc or not?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-dotnet-client/pull/2/files#r26291861>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf